apps/test-viewer/src/components/repositories-tree/FormatLabel.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ITwinRepositoryType } from "./RepositoriesType";
+
+/**
+ * @internal
+ */
+export function formatLabel(iTwinRepositoryType: ITwinRepositoryType) {
```

Comment:
maybe we just do something like this, https://stackoverflow.com/questions/26188882/split-pascal-case-in-javascript-certain-case?
convert from pascal case to sentence case?

Reply:
This looks interesting, I think it would work in majority of cases. But probably need to leave us ability to override the label with what we want, since the type won't necessarily convert to a proper label (e.g. for `iModel` I guess it might convert to "I Model"?)

Reply:
yea fair point about iModel